### ISSUE TITLE
Allow next/prev post resolver arguments

### ISSUE THREAD

**Issue by drewbaker** [funkhaus/fuxt-backend#28]
So currently we have a custom resolver to get the next post, or page in a GQL query:
https://github.com/funkhaus/fuxt-backend/blob/master/functions/gql-functions.php#L141
https://github.com/funkhaus/fuxt-backend/blob/master/functions/gql-functions.php#L198

But when it comes to "[next post](https://github.com/funkhaus/fuxt-backend/blob/master/functions/gql-functions.php#L184)" often we'd want to be able to set the params that the underlying WordPress functions allows for to get "next post in same category" : https://developer.wordpress.org/reference/functions/get_next_post/

```
$in_same_term = false
$excluded_terms = ''
$taxonomy = 'category'
```

Probably makes sense to "WP-GQL" style the naming convention here, and call it "termIn" rather than the "excluded_terms".

So I'd like to have some "resolver-arguments" for those settings:
https://www.wpgraphql.com/docs/graphql-resolvers/#resolver-arguments

So we could do this type of query on a post:

```
      nextPost(inSameTerm: true, termNotIn: '1,2,3', taxonomy: 'tag') {
        node {
          id
          title
        }
      }
```

Would be really helpful if we could allow those terms to be slugs also, so a `termSlugNotIn` as another argument that could be something like `termSlugNotIn: 'featured, uncategorized'`.

Now this it would also be really good if we could translate those same arguments for `nextPage` queries. So I could get next page that had the same tags. This isn't something there is a native WordPress function for, but you can see that it wouldn't be too hard to allow for same arguments for nextPage.
https://github.com/funkhaus/fuxt-backend/blob/master/functions/gql-functions.php#L182
https://github.com/funkhaus/fuxt-backend/blob/master/functions/gql-functions.php#L275

**Comment by rsm0128** [comment]
Created a [PR](https://github.com/funkhaus/fuxt-backend/pull/31) to fix this issue

**Comment by drewbaker** [comment]
Closed with [65497d4](https://github.com/funkhaus/fuxt-backend/commit/65497d47b6c044150bf3b6ed84a16081e9cce453)
